homeassistant/components/device_tracker/__init__.py

```diff
@@ -677,8 +678,8 @@ def async_device_tracker_scan(now: dt_util.dt.datetime):
 
             hass.async_add_job(async_see_device(**kwargs))
 
+    hass.async_add_job(async_device_tracker_scan(None))
```

Comment:
Because we're not yielding between this and the old place, this won't have any effect. Tasks only get executed when we yield.